scripts/abis/ERC20.ts

```diff
@@ -0,0 +1,224 @@
+const ERC20 = [
```

Comment:
could possibly get this from viem too

Reply:
How? Does Viem export this? I can't seem to find it anywhere.

Reply:
I feel like it *should* but also can't find it either 😞